PIL/Image.py

```diff
@@ -588,11 +588,11 @@ def _dump(self, file=None, format=None):
 
         self.load()
         if not format or format == "PPM":
-            self.im.save_ppm(file)
+            self.im.save_ppm(file, **options)
```

Comment:
Okay. Well, that's unfortunate. But again, I am not volunteering to rewrite a perfect replacement. There is a visual problem I am trying to fix.